teradatalabs/cdh5-hive-kerberized/files/supervisord.d/hive-server2.conf

```diff
@@ -0,0 +1,8 @@
+[program:hive-server2]
```

Comment:
Right, this property is set in both, but has different values in each. If you refer to the document I linked to, that's what it suggests to do -- to set the property in command line as well as in hive-site.xml.